### PR TITLE
Update dns resource group for blob private dns

### PR DESCRIPTION
**Pull request by jyothi-balla**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/EI-1481

### Change description ###
Update dns resource group for private dns

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -227,9 +220,22 @@ variable "private_endpoint_connection_name" {
...
   type        = list(string)
+  default     = []
 }
 
 variable "enable_data_lookup" {
   type    = bool
   default = false
 }
+
+variable "blob_resource_group_name" {
```
Comment (varunvijayagopal) [inline variables.tf:231]: in root module it says as string

**Review by varunvijayagopal — APPROVED** [review]

**Review by varunvijayagopal — CHANGES_REQUESTED** [review]
change

File: variables.tf (hunk 1 above)
Comment (jyothi-balla) [reply variables.tf:231]: creating this variable in the locals, so removed the var definition from root module

**Review by varunvijayagopal — APPROVED** [review]
LGTM

**Pull request merged**